vowpalwabbit/core/src/reductions/automl.cc

```diff
@@ -185,6 +188,7 @@ VW::LEARNER::base_learner* VW::reductions::automl_setup(VW::setup_base_i& stack_
                .experimental())
       .add(make_option("oracle_type", oracle_type)
```

Comment:
this case would be good to test, you would "depends_on" mechanism in https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/test/core.vwtest.json

Reply:
Added two tests which check all the overrides